frb_codegen/src/generator/dart/mod.rs

```diff
@@ -349,6 +372,12 @@ fn generate_dart_implementation_body(spec: &DartApiSpec, config: &Opts) -> Acc<D
         )
     }));
 
+    lines.push_acc(Acc {
```

Comment:
(same put to frb_dart/frb_rust)

If the port is even a field in parent class in frb_dart, then this generated class even do not need to bother this